scripts/pvscheck.sh

```diff
@@ -261,15 +263,6 @@ install_pvs() {(
   rmdir "$pvsdir"
 )}
 
-adjust_path() {
```

Comment:
It does not have to be a function (which you have to look up when reading the source), when it is only used once.